src/components/Roadmap/Roadmap.jsx

```diff
@@ -14,126 +14,23 @@ import user1Picture from "../../assets/images/user1.png";
 import user2Picture from "../../assets/images/user2.png";
 import { callOpenAiApi } from "../../utils/Functions/openaiFunctions";
 import LoadingPage from "../LoadingPage/LoadingPage";
+import { GOALS_DATA, MOCK_MATCH_DATA } from "../../mock-data/roadmapMockData";
 
 const Roadmap = () => {
   const [activeGoal, setActiveGoal] = useState(null);
   const [isModalOpen, setmodalOpen] = useState(false);
 
   const [hovering, setHovering] = useState(null);
 
-  const [user1Pic, setUser1Pic] = useState(user1Picture); //replace user1 with null instead of image placeholder
-  const [user2Pic, setUser2Pic] = useState(user2Picture); //replace user2 with null instead of image placeholder
+  const [user1Pic, setUser1Pic] = useState(null); 
```

Comment:
If you have the fallback images you can just remove lines 25 and 26 completely. There's no need for state.

Reply:
But since it is called multiple times, can't the state be used to sync every place that uses it instead of going back to find everywhere it is used later?